source/draw/gpu/opengl/OpenGLPacked.ooc

```diff
@@ -84,7 +85,7 @@ OpenGLPacked: abstract class extends GpuImage {
 					gpuMap add("texture0", image)
 			}
 		}
-		gpuMap useProgram(this, projection * view * model, textureTransform)
+		gpuMap useProgram(this, flat * projection * view * model, textureTransform)
```

Comment:
Applying a translation to a camera space point would translate in 3D with parallax.
Applying a translation to a projection matrix will not translate it, just tilt the axis system.
Others have tried this before: http://stackoverflow.com/questions/28018458/how-to-translate-the-projected-object-in-screen-in-opengl